xbmc/cores/VideoPlayer/DVDDemuxers/DVDDemuxFFmpeg.cpp

```diff
@@ -323,7 +323,6 @@ bool CDVDDemuxFFmpeg::Open(const std::shared_ptr<CDVDInputStream>& pInput, bool
     }
     if (result < 0)
     {
-      m_pFormatContext->flags |= AVFMT_FLAG_PRIV_OPT;
```

Comment:
History wise, AVFMT_FLAG_PRIV_OPT was deprecated in ffmpeg 4.4 and should have been removed in 5.0.
https://github.com/FFmpeg/FFmpeg/commit/2ff40b98ecbd9befadddc8fe665a391f99bfca32